<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(90, 115, 130);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>1051</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="rowCount">
     <number>50</number>
    </property>
    <property name="columnCount">
     <number>9</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>115</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Flight ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Leg No.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Scheduled Depar.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Scheduled Arrival</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Departure Airport</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Arrival Airport</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Seats Available</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Seats Booked</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="load_clerk">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>470</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(9, 9, 9);
color: rgb(231, 243, 255);</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insert_1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>720</y>
      <width>93</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(9, 9, 9);
color: rgb(231, 243, 255);</string>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(9, 9, 9);
color: rgb(231, 243, 255);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>1011</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Insert New Flights:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Flight ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="fid"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Airline Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="air_name"/>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Day Of Depar:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLineEdit" name="day"/>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="insert">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(9, 9, 9);
color: rgb(231, 243, 255);</string>
       </property>
       <property name="text">
        <string>Insert</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>1011</width>
      <height>123</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" rowspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Flight ID:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Insert New &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Flight Leg:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2" colspan="2">
      <widget class="QLineEdit" name="fid_1"/>
     </item>
     <item row="0" column="7" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Scheduled Departure:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" rowspan="2" colspan="2">
      <widget class="QLineEdit" name="leg_no"/>
     </item>
     <item row="1" column="7" colspan="2">
      <widget class="QDateTimeEdit" name="depar">
       <property name="maximumDate">
        <date>
         <year>2022</year>
         <month>1</month>
         <day>29</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2020</year>
         <month>12</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d h:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Scheduled Arrival:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" rowspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Leg No.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8" colspan="2">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Seats Available:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QLineEdit" name="status"/>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="froms">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>HOU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JFK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ORD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MIA</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="9" colspan="2">
      <widget class="QDateTimeEdit" name="arriv">
       <property name="maximumDate">
        <date>
         <year>2022</year>
         <month>1</month>
         <day>29</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2020</year>
         <month>12</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>yyyy-M-d h:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="to">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>HOU</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JFK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LAX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ORD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MIA</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QSpinBox" name="seats"/>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Status:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;To:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;From:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
